src/ducks/SANCharts/metrics/submetrics.js

```diff
@@ -0,0 +1,29 @@
+import { Line } from 'recharts'
+
+export const SOCIAL_TWITTER_INTERVALS = ['24h', '7d']
+
+export const TWITTER_FOLLOWERS_METRIC = {
+  queryKey: 'twitter_followers',
+  category: 'Social',
+  node: 'line',
+  Component: Line,
+  description:
+    "Shows the number changes of followers on the project's official Twitter account over time."
+}
```

Comment:
Why make it so complicated and duplicating already existing properties of `Metrics.historyTwitterData`. I've already suggested how to make it simpler in [this comment](https://github.com/santiment/app/pull/1076#discussion_r394168826)